app/recordtransfer/templates/recordtransfer/about.html

```diff
@@ -14,105 +14,34 @@
 </p>
 
 {% if FILE_UPLOAD_ENABLED %}
-    <div class="title-text margin-top-25px">{% trans "Maximum Submission Size" %}</div>
-    <p>
-        {% blocktrans %}
-        Submissions made through the transfer portal are subject to the maximum submission size.
-        You may upload a <b>maximum of {{ MAX_TOTAL_UPLOAD_COUNT }} files</b>, and <b>at most
-        {{ MAX_TOTAL_UPLOAD_SIZE_MB }} MB of files</b>. Any <b>one file may not exceed
-        {{ MAX_SINGLE_UPLOAD_SIZE_MB }} MB</b>. If you would like to submit <i>more</i> than this
-        amount, please split your submission into multiple parts, and use the group form to group similar
-        submissions together so our archivists know your submissions are related.
-        {% endblocktrans %}
-    </p>
-    <div class="title-text margin-top-25px">{% trans "Accepted File Types" %}</div>
-    <p>
-        {% blocktrans %}
-        The following are the types of files that donors can submit using this application.
-        {% endblocktrans %}
-    </p>
-    {% for group, extensions in ACCEPTED_FILE_FORMATS.items %}
-        {% if extensions %}
-            <div class="medium-text">{{ group }} Files</div>
-            <ul>
-                {% for extension in extensions %}
-                    <li>{{ extension }}</li>
-                {% endfor %}
-            </ul>
-        {% endif %}
-    {% endfor %}
-{% endif %}
-
-<h1 class="margin-top-25px" id="form-help">{% trans "Form Help" %}</h1>
+<div class="title-text margin-top-25px">{% trans "Maximum Submission Size" %}</div>
 <p>
     {% blocktrans %}
-    Refer to the following section if you come across a part of the form that you need assistance
-    with.
+    Submissions made through the transfer portal are subject to the maximum submission size.
+    You may upload a <b>maximum of {{ MAX_TOTAL_UPLOAD_COUNT }} files</b>, and <b>at most
+        {{ MAX_TOTAL_UPLOAD_SIZE_MB }} MB of files</b>. Any <b>one file may not exceed
+        {{ MAX_SINGLE_UPLOAD_SIZE_MB }} MB</b>. If you would like to submit <i>more</i> than this
+    amount, please split your submission into multiple parts, and use the group form to group similar
+    submissions together so our archivists know your submissions are related.
```

Comment:
There is extra unnecessary whitespace at the start of these lines:

```suggestion
    Submissions made through the transfer portal are subject to the maximum submission size.
    You may upload a <b>maximum of {{ MAX_TOTAL_UPLOAD_COUNT }} files</b>, and <b>at most
    {{ MAX_TOTAL_UPLOAD_SIZE_MB }} MB of files</b>. Any <b>one file may not exceed
    {{ MAX_SINGLE_UPLOAD_SIZE_MB }} MB</b>. If you would like to submit <i>more</i> than this
    amount, please split your submission into multiple parts, and use the group form to group similar
    submissions together so our archivists know your submissions are related.
```